pkg/services/cosmosdb/common-provision.go

```diff
@@ -0,0 +1,42 @@
+package cosmosdb
+
+import (
+	"errors"
+
+	"github.com/Azure/open-service-broker-azure/pkg/generate"
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+	log "github.com/Sirupsen/logrus"
+	uuid "github.com/satori/go.uuid"
+)
+
+func generateDatabaseName(location string) string {
+	databaseName := uuid.NewV4().String()
+	// CosmosDB currently limits database name to 50 characters,
```

Comment:
This actually a limitation on _account_ names, I think? Can comment and function name both be updated?

Reply:
oh good catch!